crt_portal/static/js/word_count.js

```diff
@@ -45,11 +46,29 @@ function updateWordCount (e) {
     }
 };
 
-if (textAreaElem) {
-  // Fire `updateWordCount()` on page load because textarea may have
-  // some initial content; for example if the user fills in the textarea,
-  // presses "Next" and then uses the back button to navigate back.
-  updateWordCount();
 
-  document.addEventListener('keyup', updateWordCount);
+function listenWordCount (e){
+  if (textAreaElem500.length > 0) {
+    // Fire `updateWordCount()` on page load because textarea may have
+    // some initial content; for example if the user fills in the textarea,
+    // presses "Next" and then uses the back button to navigate back.
+    // assuming only one per page
+      updateWordCount(e, max=500, textAreaElem=document.getElementById(textAreaElem500[0].id));
+  };
+  if (textAreaElem10.length > 0) {
+      updateWordCount(e, max=10, textAreaElem=document.getElementById(textAreaElem10[0].id));
+  };
 };
+
+document.addEventListener('keyup', listenWordCount);
+
+
+function checkOther(elem){
+  var y = document.querySelectorAll('.' + 'usa-checkbox__input').item(13);
```

Comment:
Let's avoid using variables named `x` and `y` in favor of named variables — clear variable names will make reading the code easier in the future for us or others.